beater/beater.go

```diff
@@ -46,8 +45,6 @@ func (bt *beater) Run(b *beat.Beat) error {
 
 	bt.server = newServer(bt.config, callback)
 	err = run(bt.server, bt.config.SSL)
-	logp.Err(err.Error())
```

Comment:
at very least log the error when the beat doesn't, inside the `if err == http.ErrServerClosed` block

Reply:
I think I miss the reason you want to log that. Isn't that the normal shutdown of the server so everything went as expected? Should this be an `Info` log that the server properly shut down?

Reply:
how do you know is expected? how do you know that accidentally someone did not send a SIGTERM to the process?
that log line is simply saying *why the server stopped. 
and again, `srv.Serve` *always returns a non nil error, you are not respecting the api.

Reply:
For me a sigterm is a proper stopping as some on purpose wanted to stop the server. If the server then shuts down properly, no Error should be logged. 

I don't think that is not respecting the API. For example `EOF` when closing a file is also an error. But often EOF is an expected error. So here we make the decision that the `ErrServerClosed` is expected.

Reply:
What I mean is that a signal can be delivered unintentionally, because eg. you give a wrong `pid` to `kill` or because you `Ctrl-C` in the wrong tab, and you might be stopping the server without noticing. The log simply says why the server stops. This is bikeshedding anyways... `Info` is good enough for me.